core-library/src/main/java/org/silverpeas/core/reminder/UserPreferenceReminderListener.java

```diff
@@ -29,6 +29,11 @@
 import org.silverpeas.core.personalization.notification.UserPreferenceEvent;
 import org.silverpeas.core.util.logging.SilverLogger;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.stream.Collectors'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)